docs/sdk-configuration.md

```diff
@@ -1,6 +1,9 @@
-> This is a design document and is not necessarily updated with evolving APIs. For up-to-date
+# 🚨 Archival Use Only! 	🚨
+> > This is a historical design document and is not updated with evolving APIs. For up-to-date
```

Comment:
```suggestion
> This is a historical design document and is not updated with evolving APIs. For up-to-date
```

Reply:
Thanks I hadn't even noticed that. Weird.